client/reader/stream/empty.jsx

```diff
@@ -34,7 +34,7 @@ class FollowingEmptyContent extends React.Component {
 					onClick={ this.recordAction }
 					href="/read/search"
 				>
-					{ this.props.translate( 'Find Sites to Follow' ) }
+					{ this.props.translate( 'Find sites to follow' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.